OLLEO-DIGITAL/Jervis-Tetch
# #17 integrate sass config

This resolves #17

----

* Updated `.gitignore` file for local vs code folder and files
* Added sveltepreprocessor
* Added config to styles path, allowing for cleaner path imports for styles
* Added `yarn` to project

## Testing steps
1. Pull down branch
2. Run `yarn` to install the latest dependencies
3. Run `yarn run dev` to start local server
4. Verify background color for `<main>` tag is red

## Screenshots
![Screenshot 2023-04-15 at 3 04 15 PM](https://user-images.githubusercontent.com/38708680/232248750-2b3cf0d7-e974-4211-99b8-1c4a92741ff3.png)
